allure-docker-kustomize-example/api/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: allure-api
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: allure-api
+    spec:
+      initContainers:
+        - name: volume-permissions
+          image: busybox
+          command: ["sh", "-c", "chown -R 1000:1000 /app/projects"]
+          volumeMounts:
+            - mountPath: /app/projects
+              name: reports-volume
+      containers:
+        - name: allure-api
+          image: allure-api-image
```

Comment:
Hi @skymoore 
Thanks for your contribution.
Quick question. How do you specify the container image and version?

Reply:
You would do that with the [images section](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/images/) in a kustomize.yaml.

In this case it would look like this:

```yaml
...
images:
  - name: allure-api-image
    newName: frankescobar/allure-docker-service
    newTag: latest
  - name: allure-ui-image
    newName: frankescobar/allure-docker-service-ui
    newTag: latest
...
```

I will update the readme to show this.  Good point.